tests/dualtor/test_orchagent_active_tor_downstream.py

```diff
@@ -78,3 +81,51 @@ def stop_garp(ptfhost):
     )
     with crm_neighbor_checker(tor), tunnel_monitor, server_traffic_monitor:
         testutils.send(ptfadapter, int(ptf_t1_intf.strip("eth")), pkt, count=10)
+
+
+def test_downstream_ecmp_nexthops(
+    ptfadapter,
+    rand_selected_dut, tbinfo,
+    toggle_all_simulator_ports,
+    tor_mux_intfs
+    ):
+    dst_server_ipv4 = "1.1.1.2"
+    nexthops_count = 4
+    set_mux_state(rand_selected_dut, tbinfo, 'active', tor_mux_intfs, toggle_all_simulator_ports)
+
+    iface_server_map = get_random_interfaces(rand_selected_dut, nexthops_count)
```

Comment:
As we discussed, not in favor of randomizing ports for each test as we may find it hard to repro failures. One iteration may fail but another may pass.

Reply:
Sure, removed randomization now.